Add-ons/UmbracoCourier/Developer/DataResolvers.md

```diff
@@ -32,20 +32,20 @@ This is what dataresolvers do, add special meaning to specific data that matches
 ## Sample Data Resolver
 To show some code as fast as possible here is a commented code sample which outlines a simple resolver:
 
-	//inherit from ItemDataResolver and implement ResolvableTypes and ShouldExecute
+	// inherit from ItemDataResolver and implement ResolvableTypes and ShouldExecute
 	public class test : ItemDataResolverProvider
 	{
-	    //resolvableTypes are the types of content which can be processed by this resolver
-	    //all built-in classes are available under Umbraco.Courier.ItemProviders
-	    //this class handles all templates and nothing else
+	    // resolvableTypes are the types of content which can be processed by this resolver
+	    // all built-in classes are available under Umbraco.Courier.ItemProviders
+	    // this class handles all templates and nothing else
 	    public override List<Type> ResolvableTypes
 	    {
 	        get { return new List<Type> { typeof(Template) }; }
 	    }
 	 
-	    //ShouldExecute, a fast way to determine if the provider should trigger or not, under a specific event
-	    //in this case the resolver will only trigger during Packaging Event and if the template has any 
-	    //JavaScript resources packaged
+	    // ShouldExecute, a fast way to determine if the provider should trigger or not, under a specific event
+	    // in this case the resolver will only trigger during Packaging Event and if the template has any 

```

Comment:
I don't think we should add spaces at the end of empty lines, do we?

Reply:
If I added space to an empty line, it would have been by mistake :)

I notice another tiny formatting issue that I've also fixed in this file.